pages: call for proposals 2025

ns-rse opened the pull request:
Closes #897

Moves the 2024 page to a new nested location and adds a 2025 page.

The 2025 page is commented out of `_data/navigation.yml` and so _doesn't_ appear in the rendered pages. To view navigate to the 2024 page `Ctrl + l` to get to the navigation bar and change `/2024/` to `/2025/` and hit `Ret`.

When page is ready to go live this will be uncommented and will also require adding to the front-page. Will address in a separate issue/pull-request.

ns-rse committed:
pages: call for proposals 2025

Closes #897

Moves the 2024 page to a new nested location and adds a 2025 page.

The 2025 page is commented out of `_data/navigation.yml` and so _doesn't_ appear in the rendered pages. To view navigate to the 2024 page `Ctrl + l` to get to the navigation bar and change `/2024/` to `/2025/` and hit `Ret`.

When page is ready to go live this will be uncommented and will also require adding to the front-page. Will address in a separate issue/pull-request.

Robadob commented on pages/collaboration/RSEtime/2025/call.md:
```diff
@@ -0,0 +1,126 @@
+
+#### Introduction
+
+The University of Sheffield recognizes the critical role that Research Software Engineers (RSEs) play in advancing
```
```suggestion
The University of Sheffield recognises the critical role that Research Software Engineers (RSEs) play in advancing
```

More common spelling in british english

jmafoster1 reviewed (approved):
Looks good to me, although we'll need to change "XX submissions" to the actual number before the page goes live.

Robadob commented on pages/collaboration/RSEtime/2025/call.md:
```diff
@@ -0,0 +1,126 @@
+industry. Areas of expertise within the group include: code optimisation and performance, reproducibility and embedding
+good software engineering practice within a project or team, developing/maintaining infrastructure, data management,
+developing computational workflows, GPU computing and Deep Learning, data science/visualisation, High Performance
+Computing using local/national/international/cloud systems, general software development, software/hardware interface,
```
```suggestion
Computing using cloud and local/national/international systems, general software development, software/hardware interface,
```

Possibly reads better this way.

ns-rse committed:
Update pages/collaboration/RSEtime/2025/call.md

Co-authored-by: Robert Chisholm <robert.chisholm@sheffield.ac.uk>

ns-rse committed:
Update pages/collaboration/RSEtime/2025/call.md

Co-authored-by: Robert Chisholm <robert.chisholm@sheffield.ac.uk>

ns-rse committed:
chore: XX > 26 in number of proposals received in 2024

Thanks @jmafoster1

ns-rse commented:
Thanks @Robadob and @jmafoster1 for the quick reviews and suggestions. Prefer british > american spelling too had skimmed over that without realsiing. I meant to lookup the number of proposals last year when making the changes but my daughter finished her karate class and we walked home.

Robadob commented on pages/collaboration/RSEtime/2025/call.md:
```diff
@@ -0,0 +1,126 @@
+To support the *One University* vision and the University research community we are pleased to announce for the second
+consecutive year the **2025 RSE collaboration call for proposals**. This initiative aims to facilitate access to RSE
+expertise for researchers who require assistance with software development, optimization, or other computational
+challenges. The 2024 call was very popular and we received XX submissions.
```
> XX submissions

todo

Robadob commented on pages/collaboration/RSEtime/2025/call.md:
```diff
@@ -0,0 +1,126 @@
+
+- **Preferred timeline:** what would be the ideal time distribution of the RSE.
+
+**The application shall be sent to the chair of the panel N. Shephard (see below for contact information) with subject
```
Are they submitting a pdf or a google form?

ns-rse replied:
Form, good spot, thanks, will update.

Robadob commented:
Possibly worth highlighting last year's winners? (e.g. to emphasise breadth/success)

ns-rse committed:
chore: clarify deadline for submissions

Romain-Thomas-Shef commented on pages/collaboration/RSEtime/2025/call.md:
```diff
@@ -0,0 +1,126 @@
+The University of Sheffield recognises the critical role that Research Software Engineers (RSEs) play in advancing
+scientific research. The University of Sheffield Research Software Engineering (RSE) team was founded in 2016 by two
+EPSRC RSE fellows to develop research software and collaborate with academics/researchers developing research
+software. The team has since grown to 11 members and has engaged with all faculties at the University.  We have worked
```
We are more than 11 now. Should probably update that number.

Romain-Thomas-Shef commented on pages/collaboration/RSEtime/2025/call.md:
```diff
@@ -0,0 +1,126 @@
+good software engineering practice within a project or team, developing/maintaining infrastructure, data management,
+developing computational workflows, GPU computing and Deep Learning, data science/visualisation, High Performance
+Computing using cloud and local/national/international systems, general software development, software/hardware interface,
+development of impact case studies, consultancy, training and education delivery and support.
```
For the context of the call we could change the list of expertise? -->

Areas of expertise within the group include: code optimisation and performance, reproducibility and embedding good software engineering practice within a project or team, developing computational workflows, GPU and High Performance Computing, Artificial intelligence/Machine Learning, data science/visualisation, general software development, software/hardware interface, development of impact case studies

Romain-Thomas-Shef commented on pages/collaboration/RSEtime/2025/call.md:
```diff
@@ -0,0 +1,126 @@
+To support the *One University* vision and the University research community we are pleased to announce for the second
+consecutive year the **2025 RSE collaboration call for proposals**. This initiative aims to facilitate access to RSE
+expertise for researchers who require assistance with software development, optimization, or other computational
+challenges. The 2024 call was very popular and we received 26 submissions.
```
Add a link to your blog post maybe?

twinkarma reviewed (approved):
Looks good @ns-rse!

ns-rse committed:
chore: PR feedback

- Correcting RSE team number.
- Clarifying team members skills and converting to list.
- Add link to blog post on last years call.

ns-rse commented:
Thanks everyone for your feedback, hopefully addressed all points. Will be publishing on Monday in case anyone _really_ wants to read through it again.

ns-rse committed:
Merge branch 'master' into ns-rse/897-proposal-call-2025
